views/api/test.py

```diff
@@ -0,0 +1,14 @@
+from django.test import TestCase, Client
+from ta_scheduler.models import User
+
+class TestSearchUserAPI(TestCase):
+    def setUp(self):
+        self.client = Client()
+        User.objects.create(username='jboy', first_name='John', last_name='Boyland', password='password')
+        User.objects.create(username='lanfar', first_name='Landon', last_name='Faris', password='password')
+
+    def test_search_user_api(self):
+        response = self.client.get('/api/search/user/?query=j')
```

Comment:
add a test for searching and getting no results

Reply:
I wanna ensure api/test.py test the /api/search/user/?query results the same thing with the UserController.searchUser. What do u think